examples/docker-compose-rlcraft.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.8"
+
+services:
+    mc:
+      image: itzg/minecraft-server:1.9.0
+      volumes:
+        - /modpacks:/modpacks
```

Comment:
I think the modpacks volume (``/modpacks:/modpacks``) contains a typo. Probalby one of eighter is meant:
a) relative path: ``./modpacks:/modpacks``
b) docker-volume: ``modpacks:/modpacks``

Reply:
@Midnight-Conqueror I agree wit @DanielRamp . Can you please adjust this?